tests/plugins/image2/manual/aspectratio.md

```diff
@@ -0,0 +1,17 @@
+@bender-tags: 4.9.0, bug, 1348
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, sourcearea, toolbar, image2, link, undo
```

Comment:
Remember to include only these plugins which are necessary to run the test, here you can go just with `wysiwygarea, toolbar, image2`.